platform/platform-filter-clientinfo/pom.xml

```diff
@@ -74,22 +74,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.00</minimum>
+                                            <minimum>0.82</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.00</minimum>
+                                            <minimum>1.00</minimum>
```

Comment:
We generally like to set these values one or two hundredths lower than observed during dev to account for platform variance.